core/iwasm/fast-jit/fe/jit_emit_conversion.c

```diff
@@ -331,6 +336,7 @@ jit_compile_op_f64_convert_i64(JitCompContext *cc, bool sign)
         JitInsn *insn;
 #if defined(BUILD_TARGET_X86_64) || defined(BUILD_TARGET_AMD_64)
         res = jit_codegen_get_hreg_by_name("xmm0_f64");
+        GEN_INSN(MOV, res, res);
```

Comment:
Should move xmm0 of f32 but not f64, as here it uses f32 xmm0 implicitly: `MOV f0, f0`, we should lock the `f0` but not `d0`?